src/test/java/nextstep/courses/domain/SessionTest.java

```diff
@@ -3,117 +3,142 @@
     void fail_paid_session_register_user_count_over() {
-        Session session = new Session(
-                normalSessionImage(),
-                normalSessionPeriod(),
-                sessionPriceOneThousand(),
-                SessionState.OPEN,
-                SessionType.PAID,
-                maxSessionUserCount()
-        );
-        assertThatThrownBy(() -> session.register(paymentOneThousand()))
+        Session session = maxAndOneThousandSession(SessionState.OPEN, SessionType.PAID);
+        assertThatThrownBy(() -> session.register(JAVAJIGI, paymentOneThousand()))
                 .isInstanceOf(SessionUserCountException.class)
                 .hasMessage("제한된 수강 신청 인원을 초과 하였습니다.");
     }
 
     @Test
     @DisplayName("성공 - 유료 강의는 지정된 수강 인원 제한을 초과하지 않았을 경우 경우 신청할 수 있다.")
     void success_paid_session_register_user_count_not_over() {
-        Session session = new Session(
-                normalSessionImage(),
-                normalSessionPeriod(),
-                sessionPriceOneThousand(),
-                SessionState.OPEN,
-                SessionType.PAID,
-                zeroSessionUserCount()
-        );
-
-        assertThatCode(() -> session.register(paymentOneThousand()))
+        Session session = zeroAndOneThousandSession(SessionState.OPEN, SessionType.PAID);
+        assertThatCode(() -> session.register(JAVAJIGI, paymentOneThousand()))
                 .doesNotThrowAnyException();
     }
 
     @Test
     @DisplayName("실패 - 유료 강의는 강의 금액과 수강료가 일치하지 않으면 수강 신청을 할 수 없다.")
     void test() {
-        Session session = new Session(
-                normalSessionImage(),
-                normalSessionPeriod(),
-                sessionPriceTenThousand(),
-                SessionState.OPEN,
-                SessionType.PAID,
-                zeroSessionUserCount()
-        );
-        assertThatThrownBy(() -> session.register(paymentOneThousand()))
+        Session session = zeroAndTenThousandSession(SessionState.OPEN, SessionType.PAID);
+        assertThatThrownBy(() -> session.register(JAVAJIGI, paymentOneThousand()))
                 .isInstanceOf(SessionPriceException.class)
                 .hasMessage("결제 금액과 강의 금액이 일치 하지 않습니다.");
     }
 
+    @Test
+    @DisplayName("성공 - 유저가 기존에 신청 하지 않은 강의일 경우 수강 신청을 할 수 있다.")
+    void success_session_register_user() {
+        Session session = zeroAndOneThousandSession(SessionState.OPEN, SessionType.PAID);
+        assertThatCode(() -> session.register(JAVAJIGI, paymentOneThousand()))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    @DisplayName("실패 - 유저가 기존에 신청한 강의일 경우 중복으로 수강 신청을 할 수 없다.")
+    void fail_session_register_user() {
+        Session session = zeroAndOneThousandSession(SessionState.OPEN, SessionType.PAID);
+        assertThatThrownBy(() -> duplicateAddSessionRegister(session))
+                .isInstanceOf(SessionUserException.class)
+                .hasMessage("강의를 이미 신청한 유저이므로 중복으로 신청할 수 없습니다.");
+    }
```

Comment:
개행을 활용해 단락을 구분하면 메서드 분리 없이도 가독성을 챙길 수 있을 것 같은데 어떻게 생각하시는지 궁금합니다
```suggestion
    @Test
    @DisplayName("실패 - 유저가 기존에 신청한 강의일 경우 중복으로 수강 신청을 할 수 없다.")
    void fail_session_register_user() {
        Session session = zeroAndOneThousandSession(SessionState.OPEN, SessionType.PAID);

        final Payment payment = paymentOneThousand();
        session.register(JAVAJIGI, payment);

        assertThatThrownBy(() -> session.register(JAVAJIGI, payment))
                .isInstanceOf(SessionUserException.class)
                .hasMessage("강의를 이미 신청한 유저이므로 중복으로 신청할 수 없습니다.");
    }
```

Reply:
좋은 것 같습니다!! 감사합니다 ㅎㅎ